install.sh

```diff
@@ -116,15 +168,21 @@ autologin-user=${USERNAME}
 autologin-session=steamos
 " > /etc/lightdm/lightdm.conf
 
+TIMEZONE="$(curl -s https://ipapi.co/timezone)"
```

Comment:
this is part of another PR. Please remove. We haven't quite decided what to do for timezone yet.